api/api_test.go

```diff
@@ -13,27 +14,48 @@ import (
 	"github.com/hashicorp/consul/testutil"
 )
 
-var consulConfig = `{
-	"ports": {
-		"dns": 19000,
-		"http": 18800,
-		"rpc": 18600,
-		"serf_lan": 18200,
-		"serf_wan": 18400,
-		"server": 18000
-	},
-	"data_dir": "%s",
-	"bootstrap": true,
-	"log_level": "debug",
-	"server": true
-}`
-
 type testServer struct {
 	pid        int
 	dataDir    string
 	configFile string
 }
 
+type TestPortConfig struct {
```

Comment:
I could have sworn that in the past the JSON encoding package was unable to export structs that were not public. Either that's changed or there's some nuance to that that I'm misremembering. I just tested it now and it works fine being private, so I'll change that.